meinberlin/react/livequestions/LikeCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import django from 'django'
+import { Pill } from '../contrib/Pill'
+import { classNames } from '../contrib/helpers'
+
+export default function LikeCard ({
+  title,
+  category,
+  likes,
+  onClickLike
+}) {
+  const likesTag = likes.session_like
+    ? django.gettext('Liked')
+    : django.gettext('Likes')
+  const addLikeTag = django.gettext('add like')
+  const undoLikeTag = django.gettext('undo like')
+
+  return (
+    <article className="modul-card card">
+      <header className="card__header">
+        <h3 className="title">{title}</h3>
+        {category && (
+          <div className="card__status status">
+            <Pill pillClass="card__pill pill pill--label">{category}</Pill>
+          </div>
+        )}
+      </header>
+      {onClickLike
+        ? (
+          <button
+            type="button"
+            className={classNames(
+              'rating-button',
+              likes.session_like && 'rating-button--active'
+            )}
+            onClick={() => onClickLike()}
+            aria-label={likes.session_like ? addLikeTag : undoLikeTag}
+          >
+            <i className="far fa-thumbs-up" />
+            {likesTag} ({likes.count})
+          </button>
+          )
+        : (
+          <span className="rating-button">
+            <i className="far fa-thumbs-up" aria-hidden="true" />
+            {likesTag} ({likes.count})
+          </span>
```

Comment:
It'd be more future proof and the translators more flexible if we used something like this
https://github.com/liqd/a4-meinberlin/blob/5706aa3fdaa7c4dab8f7c061c0e031b545c4a1ba/meinberlin/react/contrib/ControlBar.jsx#L25

Reply:
Unless I'm misunderstanding, we're not interpolating based on pluralisation though, it's either **Likes** or **Liked** based on `session_like`?

Reply:
Ahh, that makes sense, sorry.